lib/tools/xctest-commands.js

```diff
@@ -0,0 +1,51 @@
+import os from 'os';
+
+const DELIMETER = ' | ';
+
+const xctestCommands = {};
+
+/**
+ * Runs an xcuitest on the device or simulator
+ * @param {string} testRunnerBundleId The bundle id of the test runner
+ * @param {string} appUnderTestBundleId The bundle id of the app under test
+ * @param {string} xctestBundleId The bundle id of the xctest package
+ * @returns {SubProcess}
+ */
+xctestCommands.runXCUITest = async function runXCUITest (testRunnerBundleId, appUnderTestBundleId, xctestBundleId) {
+  const uiTestProcess = this.createSubProcess([
+    'xctest', 'run', 'ui', '--udid', this.udid, xctestBundleId, appUnderTestBundleId, testRunnerBundleId
```

Comment:
I would rather split args to different lines to make it easier to read, like

```
[
'xctest', 'run', 'ui',
'--udid', this.udid,
....
]
```

Reply:
👍